numpyro/distributions/constraints.py

```diff
@@ -550,13 +550,17 @@ def _transform_to_corr_cholesky(constraint):
 
 @biject_to.register(greater_than)
 def _transform_to_greater_than(constraint):
+    if constraint is positive:
```

Comment:
Is this just for efficiency, or does it lead to some corner cases in dynamic support?

Reply:
Yes, it is just for efficiency.